<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>text_book</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>350</width>
      <height>400</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="lessons">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>350</width>
        <height>390</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QListWidget" name="lessonList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>390</height>
        </rect>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="prefabs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>400</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="page3_btn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Создать запрос</string>
       </property>
      </widget>
      <widget class="QListWidget" name="prefabsList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>350</width>
         <height>360</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>400</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="prefabText">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>350</width>
         <height>350</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="submitPrefab">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Завершить создание</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="nav_menu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="page1_btn">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Учебник</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page2_btn">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>5</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Запросы</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
